games-engines/scummvm/scummvm-1.8.1.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Script Creation Utility for Maniac Mansion Virtual Machine"
+DESCRIPTION="ScummVM is a program which allows you to run certain classic \
+graphical point-and-click adventure games, provided you already have their \
+data files. The clever part about this: ScummVM just replaces the executables \
+shipped with the games, allowing you to play them on systems for which they \
+were never designed!"
+HOMEPAGE="http://scummvm.org"
+SOURCE_URI="http://scummvm.org/frs/scummvm/$portVersion/scummvm-$portVersion.tar.xz"
+CHECKSUM_SHA256="935f53d7e4c460fbc22c58fee5007d4858891e4961831f53f575a3594b7f612a"
+REVISION="1"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2001-2014 ScummVM Team"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+ADDITIONAL_FILES="scummvm.rdef"
```

Comment:
Not very important, but according to https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering ADDITIONAL_FILES should appear before ARCHITECTURES.
And COPYRIGHT, LICENSE and REVISION should appear between HOMEPAGE and SOURCE_URI.
But I think there are still many recipes whith the _old_ sorting.